### ISSUE TITLE
Solvate prefix and renaming water segids

### ISSUE THREAD

**Issue by cuzzo87** [Acellera/htmd#347]
#345 To solve this issue I think that the best approach is to renaming the default segids from:
WT0-99 to W(\*)(\*) where * = A-Z;0-9

**Comment by j3mdamas** [comment]
The first and easiest is to change the default prefix from `WT` to `W`. This will already allow 100 blocks of water to be added.

Then, the code can be changed to allow the 36 * 36 = 1296 blocks of water. Unless @stefdoerr has a better suggestion. But from what we understood from the code, we will not be able to parse the overlap with the `vmdselection` if the segid has more than 3 characters, so there seems to be no way around this (if one wants to use the `vmdselection`)

**Comment by stefdoerr** [comment]
We should not use atomselect in solvate anyway if it is avoidable. This code was ported from VMD tcl code for solvating. I will look at it when I'm back.

I agree we can make more segments

**Comment by stefdoerr** [comment]
@j3mdamas you haven't pushed your changes I assume? Could you please because I started making fixes in solvate

**Comment by j3mdamas** [comment]
I didn't make any changes related to this. What was done was changing the
default prefix at the script level, right Alberto?

**Comment by stefdoerr** [comment]
Ah ok fine. I'll modify it then

**Comment by stefdoerr** [comment]
This is actually a non-issue caused by a mistake of mine. Fixed here: https://github.com/Acellera/htmd/commit/1e0b3aab043f0ca208089da0495c9acba8b0599c

Essentially you are allowed to use 4 character segids in VMD as well. I was wrong when I wrote that catch for > 3 segids. The solvate code already has conditions for big amounts of waters in which case it switches to hex or alphanumerical for the segid names in which case you have 26+9=35 characters so 35^2=1225 water segments * 27783 atoms = 34034175 water atoms available. 

If we ever hit that limit (which I doubt with current or future GPUs) I will revise the code to use 'W' instead of 'WT' :P
